<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(161, 212, 255)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="navMyPreferences">
       <property name="text">
        <string>My Preferences</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="navFindMatchButton">
       <property name="text">
        <string>Find Match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="navManualSearchButton">
       <property name="text">
        <string>Manual Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="navMyFavoritesButton">
       <property name="text">
        <string>My Favorites</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>900</width>
       <height>555</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255)
</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Login0">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Adopet (Login Page) </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>170</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>240</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>280</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>330</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>330</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="MeetPetPage">
      <widget class="QLabel" name="petPic">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>180</y>
         <width>311</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string>picture of Pet</string>
       </property>
      </widget>
      <widget class="QLabel" name="petBio">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>340</y>
         <width>301</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Bio</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="petName">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="backButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;-- Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>70</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>56</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>â™¡</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="petAttributes">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>170</y>
         <width>341</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="bioHeader">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>300</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bio</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
